dlpy/tests/test_model.py

```diff
@@ -1039,6 +1043,422 @@ def test_mobilenetv2(self):
                                        offsets=255*[0.485, 0.456, 0.406],
                                        norm_stds=255*[0.229, 0.224, 0.225])
 
+    def test_fit_tensorboard(self):
+        try:
+            import tensorflow as tf
+            import numpy as np
+            from tensorflow.core.util import event_pb2
+        except:
+            unittest.TestCase.skipTest(self, "tensorflow and/or np not found in the libraries")
+
+        if self.data_dir is None:
+            unittest.TestCase.skipTest(self, "DLPY_DATA_DIR is not set in the environment variables")
+
+        if os.path.exists(self.data_dir + '_TB'):
```

Comment:
data_dir is in the server side - you cannot use os.path.exists (os in this case is in the client side.)